onos-config/templates/partitionset.yaml

```diff
@@ -9,11 +9,12 @@ metadata:
     release: "{{ .Release.Name }}"
     heritage: "{{ .Release.Service }}"
 spec:
-  partitions: {{ .Values.store.raft.partitions }}
+  clusters: {{ .Values.store.raft.clusters }}
   template:
     spec:
-      size: {{ .Values.store.raft.partitionSize }}
-      raft:
-        electionTimeoutMillis: 5000
-        heartbeatPeriodMillis: 1000
-{{- end }}
\ No newline at end of file
+      partitions: {{ .Values.store.raft.partitions }}
+      replicas: {{ .Values.store.raft.replicas }}
```

Comment:
`replicas` is a property of the `backend` object below and should be moved there